.github/workflows/build-windows.yml

```diff
@@ -7,23 +7,30 @@ jobs:
         shell: cmd
     strategy:
       matrix:
-          version: ['8.3']
+          version: ['8.3', '8.4']
           arch: [x64, x86]
           ts: [nts, zts]
+          exclude:
+            - { version: '8.4', arch: x64, ts: zts }
+            - { version: '8.4', arch: x86, ts: nts }
+            - { version: '8.4', arch: x86, ts: zts }
     runs-on: windows-latest
     steps:
       - name: Checkout memcached
         uses: actions/checkout@v4
       - name: Setup PHP
         id: setup-php
-        uses: php/setup-php-sdk@v0.9
+        uses: php/setup-php-sdk@v0.10
         with:
           version: ${{matrix.version}}
           arch: ${{matrix.arch}}
           ts: ${{matrix.ts}}
           deps: zlib
+          cache: true
       - name: Fetch libmemcached
-        run: curl -OLs https://windows.php.net/downloads/pecl/deps/libmemcached-1.1.1-${{steps.setup-php.outputs.vs}}-${{matrix.arch}}.zip && 7z x libmemcached-1.1.1-${{steps.setup-php.outputs.vs}}-${{matrix.arch}}.zip -o..\deps
+        run: |
+          set MEMCACHED_FILENAME=libmemcached-1.1.1-${{steps.setup-php.outputs.vs == 'vs17' && 'vs16' || steps.setup-php.outputs.vs}}-${{matrix.arch}}.zip
```

Comment:
Why 1.1.1 from 2021?
libmemcached is [currently at 1.1.4](https://github.com/awesomized/libmemcached/releases/tag/1.1.4) which actually contains a fix for a [possible security issue](https://github.com/awesomized/libmemcached/security/advisories/GHSA-wwmh-39wj-fx59).
I totally understand, though, if the answer is obviously manpower, etc...

Reply:
> Why 1.1.1 from 2021?

Must be one of the more recent dependency builds. Anyhow, let's see how https://github.com/winlibs/winlib-builder/actions/runs/11332741118 goes.

Reply:
Looks like I'll have to revert https://github.com/awesomized/libmemcached/commit/1a15a0895b863a02d598a317f4c0215edef12dc5 to get back to PHP Windows naming conventions.

Reply:
That commit looks like a fix to not end up with liblibfoo.a
What's the exact problem on your side?
I guess we could fix both sides if we get rid of the ALIAS shenanigans.

Reply:
Without the patch, on Windows memcached.lib is built; with the fix it's libmemcached.lib. The former is probably common on Windows, but the latter is the PHP convention for import libraries (would be libmemcached_a.lib for static libs). If you "fix" this, others building the library might not get the library names they want/expect.

Anyway, applying a patch is not hard, and [the build was successful](https://github.com/winlibs/winlib-builder/actions/runs/11343821419). I've built ext/memcached locally successfully against libmemcached 1.1.4.

The actual problems are lack of automation of doing and testing dependency builds, and of course, that we can hardly watch for new dependency releases (way too many packages). And some packages need way more heavy patching anyway.

Now, someone needs to upload the [libmemcached-1.1.4 builds](https://github.com/winlibs/winlib-builder/actions/runs/11343821419) to the download server. Maybe @shivammathur is in the mood. :)

Reply:
PR to use it for Windows CI: #564.

Reply:
🥳